content/metadata.yaml

```diff
@@ -411,12 +411,13 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - University of California, Berkeley
-# - name: Lawrence Hunter
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - University of Colorado
+- name: Lawrence Hunter
+   orcid: 0000-0003-1455-3370
+   github: LEHunter
+   twitter: @ProfLHunter
```

Comment:
You could consider removing the `@` character here.  The Twitter link should be functional even if you keep it though.